src/main/kotlin/com/openlattice/shuttle/destinations/PostgresDestination.kt

```diff
@@ -350,7 +345,7 @@ class PostgresDestination(
                 ps.setObject(2 + offset, version)
                 ps.setObject(3 + offset, version)
                 ps.setObject(4 + offset, entitySetId)
-                ps.setArray(5 + offset, PostgresArrays.createUuidArray(connection, entities.keys))
+                ps.setArray(5 + offset, PostgresArrays.createUuidArray(connection, entityKeyIds))
```

Comment:
use the entityKeyIdsArr that was passed in (what it used be back before lock was added in).